### ISSUE TITLE
Return value of numpy.bincount misidentified as tuple instead of ndarray

### ISSUE THREAD

**Issue by hemberger** [PyCQA/pylint#3974]
### Steps to reproduce
1. Create a file `test.py`:
```python
"""docstring"""
import numpy
numpy.bincount([1,2,3]).argmax()
```
2. Run `pylint test.py`

### Current behavior
```
~ $ pylint test.py
************* Module test
test.py:3:0: E1101: Instance of 'tuple' has no 'argmax' member (no-member)
```

### Expected behavior
No error, because `numpy.bincount` returns an `ndarray` (see https://numpy.org/doc/stable/reference/generated/numpy.bincount.html).

### pylint --version output
pylint 2.6.1-dev1
astroid 2.4.2
Python 3.7.4 (default, Aug 13 2019, 20:35:49) 
[GCC 7.3.0]

(I have tested that this error exists at least as far back as pylint 2.4.2 w/ astroid 2.3.1.)

**Comment by hippo91** [comment]
@hemberger thanks for your report. It sounds like our `astroid`'s `numpy` brain needs to be be updated.
